crates/sui-sdk/src/lib.rs

```diff
@@ -227,26 +429,35 @@ impl SuiClient {
         };
         Ok(())
     }
-}
 
-impl SuiClient {
-    pub fn transaction_builder(&self) -> &TransactionBuilder {
-        &self.transaction_builder
-    }
-    pub fn read_api(&self) -> &ReadApi {
-        &self.read_api
-    }
+    /// Returns a reference to the coin read API
     pub fn coin_read_api(&self) -> &CoinReadApi {
         &self.coin_read_api
     }
+
+    /// Returns a reference to the event API
     pub fn event_api(&self) -> &EventApi {
         &self.event_api
     }
+
+    /// Returns a reference to the governance API
+    pub fn governance_api(&self) -> &GovernanceApi {
+        &self.governance_api
+    }
+
+    /// Returns a reference to the quorum driver API
     pub fn quorum_driver_api(&self) -> &QuorumDriverApi {
         &self.quorum_driver_api
     }
-    pub fn governance_api(&self) -> &GovernanceApi {
-        &self.governance_api
+
+    /// Returns a reference to the read API
+    pub fn read_api(&self) -> &ReadApi {
+        &self.read_api
+    }
+
+    /// Returns a reference to the transaction builder API
+    pub fn transaction_builder(&self) -> &TransactionBuilder {
+        &self.transaction_builder
     }
```

Comment:
Similar comment here -- better not to have a doc comment at all than to have one that just repeats the function name.  But in these cases, maybe you can add doc comments explaining what you would use these APIs for?